src/components/OnboardingList.js

```diff
@@ -38,25 +40,30 @@ function OnboardingCheckListItem({ children, actions, state, ...props }) {
           ? { onClick: manageAction.trigger }
           : {})}
       >
-        <CheckCircleIcon
+        {!state.hideCheck && <CheckCircleIcon
           className={classNames(
             "nfd-w-[1.125rem]",
             state.isCompleted
               ? "nfd-text-[--nfd-ecomemerce-text-success]"
               : "nfd-text-[--nfd-ecommerce-text-light]"
           )}
-        />
-        <span className="nfd-flex-1 nfd-text-black">{props.text}</span>
+        />}
+        <span className="nfd-flex-1 nfd-text-black" {...(props["data-nfdhelpcenterquery"] 
+        ? {"data-nfdhelpcenterquery": props["data-nfdhelpcenterquery"]} 
+        : {}
+    )} >{props.text}</span>
         {manageAction.isMutating ? (
           <Spinner size="4" className="nfd-text-primary" />
         ) : (
-          <ArrowLongRightIcon className="nfd-text-black nfd-w-[1.125rem]" />
+          (state.showText || <ArrowLongRightIcon className="nfd-text-black nfd-w-[1.125rem]" />)
         )}
       </Link>
     </li>
   );
 }
 
+const brandName = (NewfoldRuntime?.sdk?.ecommerce?.brand_settings?.name).toLowerCase();
```

Comment:
we use `NewfoldRuntime?.plugin?.brand` we can also skip toLowerCase() if we use that variable.